src/AzDoBridge/Clients/AzureDevOpsClient.cs

```diff
@@ -8,23 +8,42 @@ namespace AzDoBridge.Clients
 {
     public class AzureDevOpsClient
     {
+        protected readonly string Username;
+        protected readonly string PersonalAccessToken;
+        Lazy<TfsTeamProjectCollection> _tfsTeamProjectCollection;
         public AzureDevOpsClient(Uri hostname, string username, string personalAccessToken)
```

Comment:
You should also add an interface and inject this to other classes/methods - not the implementation.
Otherwise it will be not possible to mock that client for tests